### PR TITLE
misc script adjustments (missing requires bad indention global variab…

### PR DESCRIPTION
**Pull request by TeoTwawki**
…les to local variables and the odd typo or 2)

Plucked from out of that other mass change they were originally included with.

### CODE REVIEW

**Comment by TeoTwawki** [comment]
This was all included in the previously merged and reverted change to `@commands`

**Comment by TeoTwawki** [comment]
..Amended to add several more missing requires I noticed a minute ago.

**Comment by TeoTwawki** [comment]
merge conflicts fixed

File: scripts/zones/Abyssea-Uleguerand/npcs/Romaa_Mihgo.lua [hunk 1]
```diff
@@ -28,15 +28,15 @@ end;
...
 
 -----------------------------------
 -- onEventFinish
 -----------------------------------
 
 function onEventFinish(player,csid,option)
---printf("CSID: %u",csid);
---printf("RESULT: %u",option);
+    -- printff("CSID: %u",csid);
+    -- printff("RESULT: %u",option);
```
Comment (takhlaq) [inline scripts/zones/Abyssea-Uleguerand/npcs/Romaa_Mihgo.lua]: printffffffffffffffffffffffffffffffff

Comment (TeoTwawki) [inline scripts/zones/Abyssea-Uleguerand/npcs/Romaa_Mihgo.lua]: fixedd

**Comment by takhlaq** [comment]
https://github.com/DarkstarProject/darkstar/pull/3304/files?w=1#diff-fec6191944475a6d7784350f8f33ec97R16
wat

**Comment by TeoTwawki** [comment]
No idea what happened there during that last rebase demo, as I never touched that line. Its fixed now anyway.

**Pull request merged**